test/framework/src/main/java/org/elasticsearch/test/CorruptionUtils.java

```diff
@@ -88,11 +88,11 @@ public static void corruptFile(Random random, Path... files) throws IOException
             // we need to add assumptions here that the checksums actually really don't match there is a small chance to get collisions
             // in the checksum which is ok though....
             StringBuilder msg = new StringBuilder();
-            msg.append("Checksum before: [").append(checksumBeforeCorruption).append("]");
+            msg.append(" before: [").append(checksumBeforeCorruption).append("]");
             msg.append(" after: [").append(checksumAfterCorruption).append("]");
             msg.append(" checksum value after corruption: ").append(actualChecksumAfterCorruption).append("]");
             msg.append(" file: ").append(fileToCorrupt.getFileName()).append(" length: ").append(dir.fileLength(fileToCorrupt.getFileName().toString()));
-            logger.info(msg.toString());
+            logger.info("Checksum info: {}", msg);
```

Comment:
You've changed the format of this slightly. Probably best to just remove `info: `.